packages/dataviews/src/view-grid.js

```diff
@@ -30,6 +30,7 @@ export default function ViewGrid( {
 	);
 	const visibleFields = fields.filter(
 		( field ) =>
+			!! field.render &&
```

Comment:
Why this change?

Reply:
Not to render anything. In current patterns page we don't display any info for that.

Reply:
I think a field without a render function is just a field that should render its value directly. And sure in the default view, we don't want to render the "sync status", but I don't see why this shouldn't be something users can display?

Reply:
Sure, I updated the code.